packages/chat-component/src/core/parser/index.ts

```diff
@@ -50,15 +52,27 @@ export async function parseStreamedMessages({
 
     // followup questions are marked either with the word 'Next Questions:' or '<<text>>' or both at the same time
     // these markers may be split across multiple chunks, so we need to buffer them!
-    // TODO: support followup questions wrapped in <<text>> markers
-    const matchedFollowupQuestionMarker = !isFollowupQuestion && chunkValue.includes('Next');
+    // TODO: remove all this logic from the frontend and implement a solution on the backend or with TypeChat
+
+    const matchedFollowupQuestionMarker =
+      (!isFollowupQuestion && chunkValue.includes('Next')) || chunkValue.includes('<<');
     if (matchedFollowupQuestionMarker) {
+      isFollowupQuestion = true;
       followupQuestionsBuffer.push(chunkValue);
       continue;
     } else if (followupQuestionsBuffer.length > 0 && chunkValue.includes('Question')) {
       isFollowupQuestion = true;
       followupQuestionsBuffer.push(chunkValue);
       continue;
+    } else if (chunkValue.includes('<<') && isFollowupQuestion) {
+      isFollowupQuestion = true;
+      chunkValue = chunkValue.replaceAll(FOLLOW_UP_QUESTION, '');
+      continue;
+    } else if (chunkValue.includes('?>') || chunkValue.includes('>')) {
+      followUpQuestionIndex = followUpQuestionIndex + 1;
+      isFollowupQuestion = true;
+      chunkValue = chunkValue.replaceAll(FOLLOW_UP_QUESTION, '');
```

Comment:
can you try removing this line call to see if you still have the same formatting?